ui/app/AppLayouts/Profile/panels/ContactPanel.qml

```diff
@@ -50,18 +50,27 @@ StatusListItem {
     signal rejectionRemoved(string publicKey)
     signal textClicked(string publicKey)
 
-    subTitle: Utils.getElidedCompressedPk(root.publicKey)
+    readonly property var d: Utils.getContactDetailsAsJson(root.publicKey)
+
+    subTitle: {
+        if (d.ensVerified) {
+            if (d.localNickname)
+                return '@' + Utils.removeStatusEns(d.name)
```

Comment:
Maybe it's better to have an util function to do this ? I am not sure if this can be used in other places. In nim there is prettyEnsName function which does it.

Reply:
Yeah I agree, although that NIM function is not exposed